rocky/rocky/views/mixins.py

```diff
@@ -111,20 +111,19 @@ def handle_connector_exception(self, exception: Exception):
         raise exception
 
     def get_observed_at(self) -> datetime:
-        if "observed_at" not in self.request.GET:
+        observed_at = self.request.GET.get("observed_at", None)
+        if not observed_at:
             return datetime.now(timezone.utc)
 
         try:
             datetime_format = "%Y-%m-%dT%H:%M:%S"
-            return convert_date_to_datetime(datetime.strptime(self.request.GET.get("observed_at"), datetime_format))
+            return convert_date_to_datetime(datetime.strptime(observed_at, datetime_format))
         except ValueError:
             try:
                 datetime_format = "%Y-%m-%d"
-                return convert_date_to_datetime(datetime.strptime(self.request.GET.get("observed_at"), datetime_format))
+                return convert_date_to_datetime(datetime.strptime(observed_at, datetime_format))
             except ValueError:
-                messages.error(
-                    self.request, _("Can not parse observed_at parameter, falling back to showing current object")
-                )
+                messages.warning(self.request, _("Can not parse date request, falling back to show current date."))
```

Comment:
This should be an error, not a warning. And if we are changing the message, I think "Can not parse date" is better than "can not parse date request" from a user perspective.